src/vtbackend/InputGenerator.cpp

```diff
@@ -70,6 +70,7 @@ bool StandardKeyboardInputGenerator::generateChar(char32_t characterEvent,
 
     if (modifiers == Modifier::Shift && characterEvent == 0x09)
     {
+        // Shift+Tab (backtab) should hit here
```

Comment:
Not just "should". 0x09 is well known ASCII code for TAB. Which makes this comment kind of pointless, don't you think so? :)

Reply:
Changed a bit, i do not recognize ASCII codes :)